### ISSUE TITLE
Toevoegen Bomen aan NLCS

### ISSUE THREAD

**Issue by ElisabethKloren** [bimloket/NLCS#112]
Er is een verkennend gesprek geweest met het normeninstituut bomen, waarbij we in ieder geval verwachten dat er zaken op elkaar afgestemd kunnen worden tussen NLCS objecten en zaken uit het datapaspoort handboek bomen. en in een later stadium mogelijk ook weer verrijkt met een informatiemodel zoals ook bijvoorbeeld bij de verkeersborden.
We hebben een Excel van het datapaspoort ontvangen, maar hier is verder nog niet veel mee gedaan.

Echter op basis van de Excel kunnen we in eerste instantie dus weer kijken welke objecten hiervan nu ontbreken in de NLCS en welke waardevol zijn om toe te voegen.
Hier zal iemand vanuit de expertcommissie, met verstand van groen, eens in moeten duiken. Echter doordat we even genoeg op het bord hebben liggen is dit er nog niet van gekomen.

Daarnaast hebben wij het, zoals destijds afgesproken, tijdens het Leveranciersoverleg aangehaald, maar hier kwamen eigen geen reacties uit waar we nu op verder door kunnen bouwen, aangezien er niet herkend werd dat er vanuit gebruikers behoefte naar was.

**Comment by ElisabethKloren** [comment]
Use case:...

**Comment by ElisabethKloren** [comment]
Ontwikkelpad: IMBOR bomen eerst opstellen, dan daarvan afleiden welke visualisaties er komen

**Comment by SanderNijhof** [comment]
Wat is hier de gedachte? De NLCS is een tekenstandaard, geen ontwerptooling. Eens met ontwikkelpad via IMBOR. IMBOR kent nog geen objecten voor groeiplaats? "graafafstand tot bomen" begrijp ik niet als NLCS object, wel als object eigenschap of als algemenen 'graaflijngrens' o.i.d.

**Comment by ElisabethKloren** [comment]
Inderdaad, ik bedoelde zoiets als een graaflijngrens; inderdaad zit dat niet in IMBOR (en wellicht komt het er ook niet in, omdat een beheerder niet degene is die die grens kan uittekenen)
en inderdaad, groeiplaatsen ede zitten nog niet in IMBOR, hier wordt op dit moment aan gewerkt.

**Comment by ElisabethDeVries** [comment]
Deze issue heeft inmiddels geleid topt het project met groeiplaatsen van bomen en kan worden afgesloten
